### PR TITLE
Added a simple theme example for labels and buttons

### PR DESCRIPTION
**Pull request by SmilyOrg**

### CODE REVIEW

File: docs/examples/FeathersSimpleThemeExample/src/FeathersSimpleThemeExample.ls [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    import loom.Application;
+    import loom2d.display.StageScaleMode;
+    import loom2d.textures.TextureSmoothing;
+    import loom2d.display.Sprite;
+    import ui.Theme;
+    import feathers.controls.Button;
+    import feathers.controls.Label;
+
+    /**
+      * Feathers can be complicated, especially with themes, so this example
```
Comment (bengarney) [inline docs/examples/FeathersSimpleThemeExample/src/FeathersSimpleThemeExample.ls]: Can you make this description the one in the summary? Or a cut down version?

**Comment by bengarney** [comment]
One nit!

**Comment by SmilyOrg** [comment]
I have to find it myself? 😛

**Comment by bengarney** [comment]
If your mouse wheel finger gets sore I can remote to your PC and help you out... :P

**Comment by SmilyOrg** [comment]
I don't think it's going to help as I won't be able to see it anyway! Maybe you forgot to submit it?

File: docs/examples/FeathersSimpleThemeExample/src/FeathersSimpleThemeExample.ls (hunk 1 above)
Comment (SmilyOrg) [inline docs/examples/FeathersSimpleThemeExample/src/FeathersSimpleThemeExample.ls]: Oh yeah, probably a good idea...

Comment (SmilyOrg) [inline docs/examples/FeathersSimpleThemeExample/src/FeathersSimpleThemeExample.ls]: Like this?

**Pull request merged**